components/time-picker/nz-time-picker.component.html

```diff
@@ -50,7 +51,9 @@
     [opened]="nzOpen"
     [nzClearText]="nzClearText"
     [nzAllowEmpty]="nzAllowEmpty"
+    (keyup.enter)="close()"
     [(ngModel)]="value">
   </nz-time-picker-panel>
+  <span tabindex="0" (focus)="close()" class="nz-tab-catching-span"></span>
```

Comment:
```suggestion
 
```
In theory the focus point should be moved to time picker panel when pressing `Tab`, it should not close the panel.

Reply:
You mean the same way as it is done in the react version: https://ant.design/components/time-picker/ ?

But I am sure this is a UX bug in that version. This behaviour would be completely unusable with a keyboard. You need to press tab about a hundred times to get anything done there. In reality, choosing time on the time-picker is not needed if the user is already using the keyboard.

Reply:
The right way is: Users use `Tab` to toggle HOUR/MINITE/SECOND/INPUT selection, use `UP_ARROW` or `DOWN_ARROW` to select number.

Reply:
Really? In order to input the time this way in the worst case the user needs 12 + 30 + 30 keystrokes.
Why would anyone do it if they can just input time with numbers, press tab and go to the next input on the form? The way you propose the user would just have to press tab 3 times more to skip through those selections.

The way I see its usage is that: 
a) there are some users who use mouse to enter time - and the time panel is great for that,
b) but there are some 'more experienced' users who want to do it faster -- and they could use keyboard for that -- 6 strokes+tab -- might take less than 2 seconds.
Imagine a receptionist somewhere who does it every day, hundreds of times.

Of course, one could implement a special form for the case b), but if the standard component could do it, why not try with a standard one?

Reply:
We need to support to cover everywhere of the panel only use keyboard, I mean use `Tab` to toggle when the panel is open, if user want to do it faster they can use 6 strokes + `Enter` + `Tab` to switch to next element, the `Tab` should not just close panel.